### ISSUE TITLE
Using PackageReference in Visual Studio

### ISSUE THREAD

**Issue by CodingWithGelos** [emgucv/emgucv#497]
I have installed Emgu.CV.runtime.windows but using NuGet's PackageReference, the DLLs such as cvextern.dll

I am unable to install Emgu.CV.runtime.windows when using the old packages.config as it says .NET framework 4.8 is not targetted.

**Comment by CodingWithGelos** [comment]
> "Packages.config" is broken for nuget package dependency resolution. You will need to upgrade "pacakges.config" to PackageRefenrece.
> See instructions to upgrade from "packages.config" to package reference here:
> [#399 (comment)](https://github.com/emgucv/emgucv/issues/399#issuecomment-767571013)
> 
> duplicate of issue #399

Hi, I am using PackageReference now and I have the runtime installed. However, when building my solution, it doesn't have any of the binary files outputted.

**Comment by 2r2d** [comment]
Use x86 or x64 as build option, AnyCPU does not copy DLL

**Comment by emgucv** [comment]
That depends on which framework you are targeting. For detail, see this:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Targeting_.Net_Framework

TL;DR
If you are targeting .Net Core or .Net 5+, you can target AnyCPU, if you are targeting .Net Framework, use either "x86" or "x64".

**Comment by albertoe** [comment]
This issue seems to occur when the following properties are used in the project file.

    <Nullable>enable</Nullable>
    <LangVersion>9.0</LangVersion>

I created a simple console app that references Emgu.CV.runtime.windows and as soon as I added the previous properties to the csproj file asper below the warning was shown.

<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
    <Nullable>enable</Nullable>
    <LangVersion>9.0</LangVersion>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Emgu.CV.runtime.windows" Version="4.5.1.4349" />
  </ItemGroup>
</Project>
